packages/environment-glimmerx/__tests__/component.test.ts

```diff
@@ -17,12 +17,6 @@ import { ComponentReturn, EmptyObject } from '@glint/template/-private/integrati
     });
   }
 
-  resolve(NoArgsComponent)({
```

Comment:
I feel like I'm missing something because it definitely seems like these should still be errors, if shaped differently. What am I not grokking?

Reply:
Covered above.